client-telemetry-reporter-plugin/kafka/README.md

```diff
@@ -0,0 +1,159 @@
+
+# How to centrally monitor Kafka clients via broker configuration
+
+Kafka's [KIP-714](https://cwiki.apache.org/confluence/display/KAFKA/KIP-714%3A+Client+metrics+and+observability)
+introduces a new capability that allows the Kafka broker to centrally track client metrics on behalf of
+applications. The broker can subsequently relay these metrics to a remote monitoring system, facilitating
+the effective monitoring of Kafka client health and the identification of any problems.
+
+The broker requires a Metrics Reporter plugin which implements the `ClientTelemetry` interface to
+send client metrics to a remote monitoring system. This tutorial demonstrates how to use the plugin
+to monitor Kafka clients.
+
+This tutorial adds `ClientOtlpMetricsReporter` that aggregates [OpenTelemetry Protocol](https://opentelemetry.io/docs/specs/otel/protocol/) (OTLP) metrics
+received from client, enhances them with additional client labels and forwards them via gRPC client
+to an external OTLP receiver. The OTLP receiver can be any OpenTelemetry compatible backend like
+Prometheus, Jaeger, Zipkin, etc. The plugin is implemented in Java and requires the jar to be
+added to the Kafka broker libs.
+
+The OTLP receiver for the tutorial is [otel-collector](https://opentelemetry.io/docs/collector/), which is a part of the OpenTelemetry project.
+The `otel-collector` receives the metrics from the Kafka broker and forwards them to a Prometheus
+instance for visualization.
+
+## Prerequisites
+
+* Java 17 or higher, e.g., follow the OpenJDK installation instructions [here](https://openjdk.org/install/) if you don't have Java.
+* Docker running via [Docker Desktop](https://docs.docker.com/desktop/) or [Docker Engine](https://docs.docker.com/engine/install/)
+
+## Setup and Cleanup
+- [Start OpenTelemetry Collector and Prometheus](#start-opentelemetry-collector-and-prometheus)
+- [Build Client Telemetry Reporter Plugin jar](#build-the-client-telemetry-reporter-plugin)
+- [Apache Kafka Broker 3.7.0 or higher](#apache-kafka-broker)
+- [Apache Kafka Client 3.7.0 or higher](#create-a-kafka-client)
+- [Cleanup](#cleanup)
+
+### Start OpenTelemetry Collector and Prometheus
+
+Clone the [confluentinc/tutorials](https://github.com/confluentinc/tutorials) GitHub repository
+(if you haven't already) and navigate to the tutorials directory:
+
+```shell
+git clone git@github.com:confluentinc/tutorials.git
+cd tutorials
+docker compose -f ./client-telemetry-reporter-plugin/kafka/docker-compose.yml up -d
+```
+
+In this example, we've configured the OTel collector (in `client-telemetry-reporter-plugin/kafka/otel-collector-config.yaml`)
+to receive OTLP metrics from the Kafka broker via gRPC on port 4317 and forwards them to `Prometheus` on port 8889.
+The configuration adds a namespace `kip-714` to the metrics which can be used to identify the source of the metrics.
```

Comment:
The relevance of `kip-714` is unclear, in particular because I think it gets translated into `kip_714` in Prometheus. So, perhaps adding "which means that the metrics in Prometheus will all start `kip_714_`".

Reply:
Done.